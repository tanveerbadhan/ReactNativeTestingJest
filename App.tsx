import React from 'react';
import { Provider } from 'react-redux';
import { PersistGate } from 'redux-persist/integration/react';
import Navigation from './src/navigation/Navigation';
import { persistor, store } from './src/redux/store';
import { Text, View } from 'react-native';

const App = () => {
  return (
    // <Provider store={store}>
    //   <PersistGate loading={null} persistor={persistor}>
    //     <Navigation />
    //   </PersistGate>
    // </Provider>
    <View>
      <Text>I m a Snapshot</Text>
    </View>
  );
};

export default App;
