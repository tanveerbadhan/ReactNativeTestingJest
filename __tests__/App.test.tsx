import { render } from '@testing-library/react-native';

import App from '../App';

test('snapshot test', () => {
    const {toJSON} = render(<App />);
    expect(toJSON()).toMatchSnapshot();
})