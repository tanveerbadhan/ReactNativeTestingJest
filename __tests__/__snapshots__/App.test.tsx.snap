// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<View>
  <Text>
    I m a Snapshot
  </Text>
</View>
`;
